UnityProject/Assets/Scripts/Doors/DoorController.cs

```diff
@@ -292,6 +298,14 @@ private void SyncIsWelded(bool _wasWelded, bool _isWelded)
 				weldOverlay.sprite = isWelded ? weldSprite : null;
 			}
 		}
+		public void ServerDisassemble(HandApply interaction)
+		{
+			Vector3Int position = Vector3Int.RoundToInt(transform.localPosition);
+			// tileChangeManager.RemoveTile(position, LayerType.Walls); Disabled until I can figure out
+			// tileChangeManager.SubsystemManager.UpdateAt(position); why removing a passable wall deletes atmos
+			Spawn.ServerPrefab(CommonPrefabs.Instance.Metal, registerTile.WorldPositionServer, count: 4);
+			GetComponent<CustomNetTransform>().DisappearFromWorldServer();
+		}
```

Comment:
DisappearFromWorld server doesn't despawn prefab but moves it to hiddenpos forever. that and #1902 probably make that nasty no pressure effect when combined.
use this:
```suggestion
        public void ServerDisassemble(HandApply interaction)
        {
            tileChangeManager.RemoveTile(registerTile.LocalPositionServer, LayerType.Walls);
            Spawn.ServerPrefab(CommonPrefabs.Instance.Metal, registerTile.WorldPositionServer, count: 4);
            Despawn.ServerSingle(gameObject);
        }
```
but! keep in mind that if false wall will be destroyed due to shooting, `            tileChangeManager.RemoveTile(registerTile.LocalPositionServer, LayerType.Walls);` won't be called, see https://github.com/unitystation/unitystation/blob/26339d2153e59fbed87b06fe43bf438a8ab1ccb1/UnityProject/Assets/Scripts/Tilemaps/Behaviours/Objects/RegisterDoor.cs#L52-L57